packages/docusaurus-theme-bootstrap/src/theme/DebugLayout/index.tsx

```diff
@@ -21,15 +21,8 @@ function Layout(props: Props): JSX.Element {
   const {siteConfig} = useDocusaurusContext();
   const {favicon, url: siteUrl} = siteConfig;
   const {image: defaultImage, metadatas} = useThemeConfig();
-  const {
-    children,
-    title,
-    noFooter,
-    description,
-    image,
-    keywords,
-    permalink,
-  } = props;
+  const {children, title, noFooter, description, image, keywords, permalink} =
+    props;
```

Comment:
Honestly I hardly find this more appealing than the previous behavior, so I'm fine with pinning Prettier version; still there are other actual style improvements as well, like removing the brackets when asserting types twice